### ISSUE TITLE
Sorting by score in searchService.cfc not working 

### ISSUE THREAD

**Issue by JonnyHayes** [blueriver/MuraSolr#13]
Hi - Think I've uncovered a bug - not sure if it's related to CF10/MSSQL or and oddity with QoQ in CF10,  or something else (or even if it is indeed a bug):

Try this at ln 403: cfquery name="rsScore" dbtype="query"
            select  contentID,score ,context from rsDbSearch
            where contentID=<cfqueryparam cfsqltype="cf_sql_varchar" value="#rs.contentID#">
        /cfquery
        < cfdump var="#rsScore#" >

and  add this:

```
SCORE:<cfdump var="#rs#"><cfabort>
<cfreturn rs />
```

The rsScore dump dumps the correct score (0.3227 etc) , rs dumps 0.

Basically - it returns the '0' value set in the rs query at line 300 and NOT the correct score.SO... upshot is that it always returns '0' instead of the correct querysetcell at ln 408. 

For my test search I used a quoted phrase - e.g. "My search term" .
Still trying to fix it from my end - if I come up with a solution before you do I'll share it here....
cheers
jon
